sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -415,17 +385,16 @@ class Explorer extends Editor<ExplorerProps, ExplorerState> {
         );
     };
 
-    private handleDevToolsToggle = (
-        e: React.MouseEvent<HTMLButtonElement>,
-        props: ActionToggleProps
-    ): void => {
-        this.maxViewerHeight = !props.selected
+    private handleDevToolsToggle = (): void => {
+        const selected: boolean = !this.state.devToolsVisible;
+        this.maxViewerHeight = selected
             ? this.maxViewerHeight / 2
             : this.maxViewerHeight * 2;
 
         this.setState({
-            devToolsVisible: !props.selected,
+            devToolsVisible: selected,
         });
+        this.setViewerToFullSize();
```

Comment:
I believe it might be safer to use the second `setState` arg as this will ensure that the function is fired after `setState` has been completed.

```suggestion
        }, this.setViewerToFullSize);
```

Reply:
Done